lib/services/northBound/contextServerUtils.js

```diff
@@ -41,8 +41,8 @@ const commands = require('../commands/commandService');
  * @return {String}                 The Tenant decribed in the request headers
```

Comment:
An entry in the CHANGES_NEXT_RELEASE file corresponding to the fix done in this PR should be included.

Reply:
Just updated it

Reply:
Fixed in [842af08](https://github.com/telefonicaid/iotagent-node-lib/pull/1249/commits/842af087eb7ca4ba342b62958345643c9a0cf52e)